modules/ROOT/examples/live-demos/url-dialog/index.js

```diff
@@ -2,17 +2,15 @@ tinymce.init({
   selector: 'textarea#url-dialog',
   toolbar: 'urldialog',
   height: 300,
-  setup: function (editor) {
+  setup: (editor) => {
     editor.ui.registry.addButton('urldialog', {
       icon: 'code-sample',
-      onAction: function () {
-        editor.windowManager.openUrl({
-          title: 'URL Dialog Demo',
-          url: '../../demo/url-dialog-demo/external-page.html',
-          height: 640,
-          width: 640
-        });
-      }
+      onAction: () => editor.windowManager.openUrl({
+        title: 'URL Dialog Demo',
+        url: '{{attachmentsdir}}/url-dialog-demo/external-page.html',
```

Comment:
Confirm this like should be different to the `url: 'external-page.html',` used above

Reply:
It should not be, as this is an internal link the one in `example.js` is as the name suggests an example and should not include the internal path.